ERROR - 2023-12-28 07:17:00 --> mysqli_sql_exception: Cannot insert duplicate record. Record with the same particulars already exists. in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `in...', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `in...')
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `in...')
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `in...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\dilg\backend\backend\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\dilg\backend\backend\app\Controllers\DatabasePPEController.php(177): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\DatabasePPEController->saveInventory()
#10 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
#11 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-12-28 07:17:00 --> Cannot insert duplicate record. Record with the same particulars already exists.
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `inventoryppe` (`entityname`, `particulars`, `classification`, `quantity`, `arrival`) VALUES (:entityname:, :particulars:, :classification:, :quantity:, :arrival:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\DatabasePPEController.php(177): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\DatabasePPEController->saveInventory()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
ERROR - 2023-12-28 07:29:29 --> mysqli_sql_exception: Cannot insert duplicate record. Record with the same particulars already exists. in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `in...', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `in...')
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `in...')
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `in...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\dilg\backend\backend\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\dilg\backend\backend\app\Controllers\DatabasePPEController.php(177): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\DatabasePPEController->saveInventory()
#10 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
#11 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-12-28 07:29:29 --> Cannot insert duplicate record. Record with the same particulars already exists.
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `inventoryppe` (`entityname`, `particulars`, `classification`, `quantity`, `arrival`) VALUES (:entityname:, :particulars:, :classification:, :quantity:, :arrival:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\DatabasePPEController.php(177): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\DatabasePPEController->saveInventory()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
